### PR TITLE
Use BOM for dependency management

### PR DESCRIPTION
**Pull request by MinnDevelopment**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,33 +86,48 @@ dependencies {
...
 
 **Gradle**
 
 ```gradle
 dependencies {
-    compile ('club.minnced:opus-java:%VERSION%@pom') {
-        transitive = true // gradle defaults to transitive false with @pom
-    }
+    compile ("club.minnced:opus-java-bom:%VERSION%")
+    compile ("club.minnced:opus-java-api")
```
Comment (kantenkugel) [inline README.md:103]: Version determined by the bom? Or how does Gradle handle those artifacts without version

File: README.md [hunk 2]
```diff
@@ -86,33 +86,48 @@ dependencies {
...
+            <type>pom</type>
+            <scope>import</scope>
+        </dependency>
+    </dependencies>
+</dependencyManagement>
+<dependencies>
+    <dependency>
+        <groupId>club.minnced</groupId>
+        <artifactId>opus-java-api</artifactId>
+    </dependency>
```
Comment (kantenkugel) [inline README.md:126]: Same here i guess

**Comment by kantenkugel** [comment]
Oh nvm... its already closed :)

File: README.md (hunk 1 above)
Comment (MinnDevelopment) [reply README.md:103]: The bom is used to defined the versions of the dependencies and compile applies them

**Pull request closed without merging**